### PR TITLE
Changed readme

### PR DESCRIPTION
**Pull request by KugelPfusch**
Updated and improved readme

Please add hacktoberfest-accepted label and merge

### CODE REVIEW

**Review by Samuuuh — CHANGES_REQUESTED** [review]
Can you summarize the contributors part and add some badges too?

**Comment by KugelPfusch** [comment]
Which badges do you want me to add and where?
What do you mean by "summarize the contributors part"?

**Comment by Samuuuh** [comment]
Remove some steps like 3. Work - commit - repeat ( be sure to be in your branch )

**Comment by KugelPfusch** [comment]
hope its fine now 😅
